README.md

```diff
@@ -157,7 +157,7 @@ github.authorization.create({
 
 ## DEBUG
 
-Set `DEBUG=node-github:*` for additional debug logs.
+Set `DEBUG=node-github` for additional debug logs.
```

Comment:
Oh good catch! Can we change it to `DEBUG=node-github*`, without the colon? That’ll make it future proof

Reply:
Sure!